src/components/steps/general-info/Funders.tsx

```diff
@@ -15,20 +15,20 @@ interface FundersProps {
     onChange: (project: Project) => void;
 }
 
+interface Funder {
+    id: string;
+    displayName: string;
+    shortName: string;
+}
+
 type ModelCollectionField = "funders";
 type AdditionalDesignationField = "additional";
 type Option = { value: string; text: string; code: string };
 
 const Funders: React.FC<FundersProps> = ({ project, onChange }) => {
     const [isDialogOpen, setDialogOpen] = React.useState(false);
-    const [automaticFill, setAutomaticFill] = React.useState(true);
-    const [value, setValue] = React.useState<
-        {
-            id: string;
-            displayName: string;
-            shortName: string;
-        }[]
-    >([]);
+    const [fundersAdded, setFundersAdded] = React.useState<Funder[]>([]);
+    const [code, setCode] = React.useState<string[]>([]);
 
```

Comment:
Note: I had also doubts with this, but after checking with Adrian, we want to set all the selected funder codes, not the last selected. So it's all very simple:

```
    const [isDialogOpen, setDialogOpen] = React.useState(false);
    const [fundersAdded, setFundersAdded] = React.useState<Funder[]>([]);
    const [code, setCode] = React.useState<string[]>([]);
```

We only need the first. The other two are not really state of the component.

```
    const [isDialogOpen, setDialogOpen] = React.useState(false);
```